cpp/util/libevent_wrapper.cc

```diff
@@ -21,6 +27,19 @@ using util::TaskHold;
 
 namespace {
 
+static Counter<string>* total_http_server_requests(
+    Counter<string>::New("total_http_server_requests", "path",
+                         "Total number of HTTP requests receieved for a given "
```

Comment:
Had a surplus of the letter 'e', thought I might be able to hide an extra one in there.